config/samples/misf_v1alpha1_managedfusionoffering.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: misf.ibm.com/v1alpha1
+kind: ManagedFusionOffering
+metadata:
+  name: managedfusionoffering-sample
+spec:
+  # TODO(user): Add fields here
```

Comment:
If we have an example doc, let's fill it in with an example

Reply:
Updated